docs/source/introduction.md

```diff
@@ -18,13 +18,23 @@ For other models, there is comprehensive documentation to inspire upon the chang
 5. Patcher modules to map weights of original model's operations to updated model's operations
 6. Exporter module to export the model source into a `ONNX` Graph.
 7. Sample example applications and demo notebooks
-8. Unit test templates. 
+8. Unit test templates.
 
 ***Latest news*** : <br>
 
-- [coming soon] Support for more popular [models](https://quic.github.io/efficient-transformers/source/validate.html#models-coming-soon) and inference optimization technique speculative decoding <br>
```

Comment:
why removed?

Reply:
I just wanted to remove the SPD. Okay, I will add the coming soon model..